src/dropdowns/dropdown-item.ts

```diff
@@ -0,0 +1,57 @@
+import {Directive, Input, HostBinding, HostListener, ElementRef} from 'angular2/core';
+import {NglDropdown} from './dropdown';
+
+@Directive({
+  selector: '[nglDropdownItem]',
+  host: {
+    'tabindex': '0',
+  },
+})
+export class NglDropdownItem {
+  @Input('nglDropdownItem') value: any;
+  @HostBinding('class.slds-is-selected') get _isSelectedClass() {
+    return this.isSelected;
+  }
+  @HostBinding('attr.aria-selected') get _isSelectedAria() {
+    return this.isSelected.toString();
+  }
+
+  private isSelected = false;
+  private dropdown: NglDropdown;
+
+  @HostListener('click') toggleSelected() {
+    this._toggleSelected();
+  }
+  @HostListener('keydown.space', ['$event'])
+  @HostListener('keydown.enter', ['$event'])
+  onKeydownToggle($event: Event) {
+    $event.preventDefault();
+    this._toggleSelected();
+  }
+
+  constructor(private elementRef: ElementRef) {}
+
+  registerParent(dropdown: NglDropdown) {
```

Comment:
Here, we need to enumerate the children so that we can chose the next focusable so it seems that in the end, `@ContentChildren` is more appropriate. We have not event streams to handle here because each item is not aware of it's siblings which is a good thing.